monai/transforms/compose.py

```diff
@@ -72,8 +72,20 @@ def __call__(self, data: Any):
 
 class Randomizable(ABC):
     """
-    An interface for handling local numpy random state.
-    this is mainly for randomized data augmentation transforms.
+    An interface for handling numpy random state locally based on a class variable `R`,
+    which is an instance of `np.random.RandomState()`.
```

Comment:
could you help update the first sentence to "An interface for handling random state locally, currently based on a class variable `R`, which is an instance of `np.random.RandomState`."

because we might consider to include the pytorch's random generators soon https://github.com/pytorch/pytorch/issues/17079

Reply:
updated in the latest commit.
Thanks.